src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -742,6 +742,15 @@ public ResponseEntity<List<ReporterModel>> getNodeReport(@Parameter(description
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studyService.getNodeReport(studyUuid, nodeUuid, nodeOnlyReport));
     }
 
+    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/modifications", produces = MediaType.TEXT_PLAIN_VALUE)
+    @Operation(summary = "Get node network modifications")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The node network modifications"), @ApiResponse(responseCode = "404", description = "The study/node is not found")})
+    public ResponseEntity<String> getNodeNetworkModifications(@Parameter(description = "Study uuid") @PathVariable("studyUuid") UUID studyUuid,
+                                                              @Parameter(description = "Node uuid") @PathVariable("nodeUuid") UUID nodeUuid) {
+        // Return json string because modification dtos are not available here
+        return ResponseEntity.ok().contentType(MediaType.TEXT_PLAIN).body(networkModificationTreeService.getNetworkModifications(studyUuid, nodeUuid));
```

Comment:
Why not using `MediaType.APPLICATION_JSON` instead of `MediaType.TEXT_PLAIN` ?

Reply:
See the comment above